src/PlusWidgets/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ SET(${PROJECT_NAME}_INCLUDE_DIRS
   ${CMAKE_CURRENT_BINARY_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}
   )
-IF(CMAKE_VERSION VERSION_GREATER_EQUAL "3.8")
+# If CMake 3.7 is the minimum version then the next line can be

```

Comment:
Is there anything that is not in 3.6? Just asking because Linux distros etc. are typically lagging behind. If we require 3.7.2 then users must download&install CMake from the website instead of just use what's installed already.